third_party/move/move-compiler-v2/transactional-tests/tests/tests.rs

```diff
@@ -128,12 +157,21 @@ fn main() {
         .collect_vec();
     let reqs = TEST_CONFIGS
         .iter()
-        .map(|c| {
+        .map(|config| {
+            let pattern = files
+                .iter()
+                .filter(|file| {
+                    (config.include.is_empty()
+                        || config.include.iter().any(|include| file.contains(include)))
+                        && (!config.exclude.iter().any(|exclude| file.contains(exclude)))
```

Comment:
Is this indentation right?  It makes it look like `(X || Y && Z)` but I'm pretty sure the parens are `((X || Y) && Z)`.  I would have hoped that rustfmt would do better.